doc/release-notes/Makefile.am

```diff
@@ -10,6 +10,12 @@ if ENABLE_RELEASE_DOCS
 
 DOC= release-$(SQUID_RELEASE)
 
+%.sgml: %.sgml.in
+	sed \
+        -e "s%[@]SQUID_VERSION[@]%$(VERSION)%g" \
+        -e "s%[@]SQUID_RELEASE[@]%$(SQUID_RELEASE)%g" \
+        < $< >$@
```

Comment:
To detect macro typos and such, please add a check that the processed/created file has no `@SQUID` expressions left.

Reply:
Done in [c19fb1d](https://github.com/squid-cache/squid/pull/1264/commits/c19fb1d9f9613b5e02d85088cf17a258865aec24)